safekeeper/src/send_wal.rs

```diff
@@ -551,7 +551,7 @@ impl<IO: AsyncRead + AsyncWrite + Unpin> WalSender<'_, IO> {
 
             self.pgb
                 .write_message(&BeMessage::KeepAlive(WalSndKeepAlive {
-                    sent_ptr: self.end_pos.0,
+                    sent_ptr: self.start_pos.0,
```

Comment:
Let's add a comment that `self.start_pos.0` is equal to `commit_lsn` in this case.